src/React/Basic/Emotion.js

```diff
@@ -34,3 +34,5 @@ exports.elementKeyed_ = (component, props) =>
 exports.global = Emotion.Global;
 
 exports.css = _homogeneousDict => Emotion.css;
+
+exports.important = prop => typeof prop === "string" ? prop + " !important" : prop;
```

Comment:
Since all base combinators for creating a single-valued `StyleProperty` now produce strings as the underlying representation we can make this `important` combinator try to handle its input as a string.